doc/source/ray-core/objects/serialization.rst

```diff
@@ -187,8 +206,8 @@ Below, we demonstrate this behavior on a function with a non-serializable object
 
 The resulting output is:
 
-
-.. code-block:: bash
+.. testoutput::
+    :options: +SKIP
```

Comment:
Instead of skipping this output, would it make sense to use `+ELLIPSIS` on the memory addresses?